src/components/Editor/EditorMenu.js

```diff
@@ -85,12 +87,13 @@ function getMenuItems(
   );
 
   const isOriginal = isOriginalId(selectedLocation.sourceId);
+  const isGenerated = isGeneratedId(selectedLocation.sourceId);
   const hasSourceMap = selectedSource.get("sourceMapURL");
   const isPrettyPrinted = isPretty(selectedSource.toJS());
 
   const jumpLabel = {
     accesskey: L10N.getStr("editor.jumpToMappedLocation1.accesskey"),
-    disabled: isGeneratedId && !hasSourceMap,
+    disabled: isGenerated && !hasSourceMap && !hasPrettyPrint,
```

Comment:
Maybe we also check `isPrettyPrinted` since we have it anyways
i.e `disabled: isGenerated && !hasSourceMap && (!hasPrettyPrint || !isPrettyPrinted),`

Reply:
yep, i like naming these bools:

isPretty = hasPrettyPrint || !isPrettyPrinted
noMap = isGenerated && !hasSourceMap

or something like that. noMap is clearly a bad name

Reply:
i changed the code to include both comments. please check